kubernetes-addons/argocd/argocd-application/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Application Name
+name: ""
+
+# The ArgoCD Project the Application belongs to.
+project: "default"
+
+# Source config for the Application
+source: 
+  
+  # Git Repo the Application points to. 
+  repoUrl: ""
+  
+  # Target revision for the repo. 
+  targetRevision: "HEAD"
+
+  # Path in the repo Argo should look for manifests. 
+  path: ""
+
+  # Helm configuration. 
+  helm :
+    values: ""
+
+# Destination cluster.
+destination:
+  server: "https://kubernetes.default.svc"
```

Comment:
At some point in the near future we have to make this a property as well as customers may employ the management cluster approach.